azure-functions/src/durable.rs

```diff
@@ -14,35 +14,36 @@ use std::{
 
 mod actions;
 mod activity_output;
-mod creation_urls;
 mod history;
-mod management_urls;
 mod orchestration_output;
 
-pub use actions::*;
-pub use creation_urls::*;
-pub use history::*;
-pub use management_urls::*;
-pub use orchestration_output::*;
-
+pub use self::actions::*;
 pub use self::activity_output::*;
-pub use self::creation_urls::*;
-pub use self::management_urls::*;
+pub use self::history::*;
 pub use self::orchestration_output::*;
 
 #[doc(hidden)]
 #[derive(Debug, Serialize, Default)]
+#[serde(rename_all = "camelCase")]
 pub struct ExecutionResult {
-    done: bool,
-    actions: Vec<Action>,
+    is_done: bool,
+    actions: Vec<Vec<Action>>,
```

Comment:
Did I miss nested actions in specs?

Reply:
It's a list of list of actions, where each entry is the actions that were created for a particular invocation of the orchestration function.  Wherever a future (action, join, or select) returns `Poll::Ready` (i.e. immediately following an `await`) is a place where a new list of actions can be built up.

Reply:
Ok. Got it. From reading the initial spec, I thought it's just a plain list of actions.